go/plugins/dotprompt/picoschema_test.go

```diff
@@ -15,158 +15,87 @@
-			        "description": "the description, which has, multiple commas"
-			}`,
-		},
-		{
-			description: "simple scalar, with description (extra whitespace)",
-			yaml:        `schema: number,    the description`,
-			want:        `{ "type": "number", "description": "the description" }`,
-		},
-		{
-			description: "simple object",
-			yaml: `schema:
-  field1: boolean
-  field2: string`,
-			want: `{
-			        "type": "object",
-			        "properties": {
-			          "field1": { "type": "boolean" },
-			          "field2": { "type": "string" }
-			        },
-			        "required": ["field1", "field2"]
-			}`,
-		},
-		{
-			description: "required field",
-			yaml: `schema:
-  req: string, required field
-  nonreq?: boolean, optional field`,
-			want: `{
-			        "type": "object",
-			        "properties": {
-			          "req": { "type": "string", "description": "required field" },
-			          "nonreq": { "type": "boolean", "description": "optional field" }
-			        },
-			        "required": ["req"]
-			}`,
-		},
-		{
-			description: "array of scalars, with and without description",
-			yaml: `schema:
-  tags(array, list of tags): string, the tag
-  vector(array): number`,
-			want: `{
-			        "type": "object",
-			        "properties": {
-			          "tags": {
-			            "type": "array",
-			            "description": "list of tags",
-			            "items": { "type": "string", "description": "the tag" }
-			          },
-			          "vector": { "type": "array", "items": { "type": "number" } }
-			        },
-			        "required": ["tags", "vector"]
-			}`,
-		},
-		{
-			description: "nested object in array and out",
-			yaml: `schema:
-  obj?(object, a nested object):
-    nest1?: string
-  arr(array, array of objects):
-    nest2?: boolean`,
-			want: `{
-			        "type": "object",
-			        "properties": {
-			          "obj": {
-			            "type": "object",
-			            "description": "a nested object",
-			            "properties": { "nest1": { "type": "string" } }
-			          },
-			          "arr": {
-			            "type": "array",
-			            "description": "array of objects",
-			            "items": {
-			              "type": "object",
-			              "properties": { "nest2": { "type": "boolean" } }
-			            }
-			          }
-			        },
-			        "required": ["arr"]
-			}`,
-		},
-		{
-			description: "simple json schema type",
-			yaml: `schema:
-  type: string`,
-			want: `{ "type": "string" }`,
-		},
-		{
-			description: "simple json schema type",
-			yaml: `schema:
-  type: string`,
-			want: `{ "type": "string" }`,
-		},
-		{
-			description: "inferred json schema from properties",
-			yaml: `schema:
-  properties:
-    foo: {type: string}`,
-			want: `{ "type": "object", "properties": { "foo": { "type": "string" } } }`,
-		},
+	type test struct {
+		Description string
+		YAML        string
+		Want        map[string]any
+	}
+
+	convertSchema(&jsonschema.Schema{})
```

Comment:
Why call `convertSchema` here and ignore the result?

Reply:
Oops, leftover debugging line.